youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/YouboraPlugin.java

```diff
@@ -133,6 +134,12 @@ private void loadPlugin() {
         });
     }
 
+    private void updateStreamingProtocol(PlayerEvent.SourceSelected sourceSelectedEvent) {
+        if (sourceSelectedEvent != null && sourceSelectedEvent.source != null && sourceSelectedEvent.source.getMediaFormat() != null) {
```

Comment:
We need to add container format (KEY_CONTENT_ENCODING_CONTAINER_FORMAT) only in case if it not populated by the app in the 'Options'.

Reply:
removed that addeded it only to streaming protocol